packages/forma-36-react-components/src/components/Skeleton/SkeletonContainer/SkeletonContainer.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+import styles from './SkeletonContainer.css';
+
+class SkeletonContainer extends React.Component {
+  static propTypes = {
+    extraClassNames: PropTypes.string,
+    children: PropTypes.node.isRequired,
+    testId: PropTypes.string,
+    ariaLabel: PropTypes.string,
+    width: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+    height: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+    preserveAspectRatio: PropTypes.string,
+    clipId: PropTypes.string,
+    gradientId: PropTypes.string,
+    primaryColor: PropTypes.string,
```

Comment:
I think instead of `primaryColor`/`secondaryColor` we should use `backgroundColor`/`highlightColor`, or `backgroundColor`/`foregroundColor`. Makes the application of the colour clearer to me